packages/ui/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "build": "npm-run-all build:css build:prefix build:src build:optimize build:package",
-    "build:css": "sass --source-map --precision=8 src/main.scss ../../dist/clr-ui/clr-ui.css; sass --source-map --precision=8 src/dark-theme.scss ../../dist/clr-ui/clr-ui-dark.css",
+    "build:css": "sass --source-map --precision=8 --watch src src/main.scss ../../dist/clr-ui/clr-ui.css; sass --source-map --precision=8 src/dark-theme.scss ../../dist/clr-ui/clr-ui-dark.css",
```

Comment:
Remove, don't want to watch by default build.

Reply:
Reverted.